setup.py

```diff
@@ -19,19 +22,13 @@
         'Programming Language :: Python',
         'Framework :: Django',
     ],
-    packages=find_packages(exclude=["tests"]),
-    dependency_links=[
-    ],
+    packages=find_packages(exclude=['tests']),
     install_requires=[
-        "django>=1.8",
-        "django-model-utils==1.4.0",
+        'django>=1.8,<1.9',
+        'django-model-utils>=1.4.0,<1.5.0',
+        'djangorestframework>=3.2.0,<3.4.0',
+        'djangorestframework-jwt>=1.6.0,<=1.7.2',
+        'edx-opaque-keys>=0.1.2,<=0.2.0',
```

Comment:
`<1.0.0` was correct here. We follow [semantic versioning](http://semver.org/), so all minor and patch changes should be compatible.